src/components/common/DesktopNavbar.tsx

```diff
@@ -184,6 +184,47 @@ export const DesktopNavbar: Component<{ menu: RouteMenu }> = ({ menu }) => {
               </Link>
             );
           }
+          if (isSuccess && menuItem.type === 'logout-btn') {
+            return (
+              <Tooltip
+                key={idx}
+                open={openPopoverAvatar}
+                handler={setOpenPopoverAvatar}
+                {...triggersAvatar}
+                className='bg-white'
+                placement='bottom-end'
+                content={
+                  <div className='p-1 shadow-md'>
+                    <div className='flex items-center gap-2'>
+                      <Avatar
+                        src='https://docs.material-tailwind.com/img/face-2.jpg'
+                        alt='avatar'
+                      />
+                      <div>
+                        <Typography variant='h6' color='gray'>
+                          Tania Andrew
+                        </Typography>
+                        <Typography variant='small' color='gray' className='font-normal'>
+                          {data ? data.email : ''}
+                        </Typography>
+                      </div>
+                    </div>
+                    <hr className='my-1' />
+                    <List className='p-0'>
+                      <ListItem onClick={() => emitEvent('logout')}>{menuItem.name}</ListItem>
```

Comment:
Duplicate code. `emitEvent` was used in `App.tsx`. Use attribute `onClick` of `menuItem` instead.

Reply:
![image](https://github.com/ngyngcphu/tick3d-fe/assets/109587630/97015e11-db95-4f52-96fc-d183642b9c39)
I test with <ListItem onClick={menuItem.onClick}>{menuItem.name}</ListItem> but it didn't work!

Reply:
Why? I tested it and it worked.
![image](https://github.com/ngyngcphu/tick3d-fe/assets/111476687/1bc5ace2-7562-4a3e-9fb7-3d49e4abd021)